thehive/app/org/thp/thehive/models/TheHiveSchemaDefinition.scala

```diff
@@ -89,6 +89,14 @@ class TheHiveSchemaDefinition @Inject() extends Schema with UpdatableSchema {
       traversal.outE[ShareTask].raw.property("actionRequired", false).iterate()
       Success(())
     }
+    .updateGraph("Add managePattern permission to admin profile", "Profile") { traversal =>
+      traversal.unsafeHas("name", "admin").raw.property("permissions", "managePattern").iterate()
+      Success(())
+    }
+    .updateGraph("Add manageProcedure permission to admin profile", "Profile") { traversal =>
```

Comment:
The permission manageProcedure must be added to orgAdmin and analyst profiles.
`Profile.analyst` must also be updated